features/s3/postObject.feature

```diff
@@ -1,9 +1,19 @@
 @s3 @integ
 Feature: POST object uploads
 
-    Scenario: Upload and object via POST
-        Given I have an "s3" client
-        And I have a file
-        When I create a POST object
-        And I make a HTTP POST request
-        Then a file is uploaded
+    Scenario: Upload an object via POST object with Signature V2
+        Given I have an s3 client and I have a file
+        When I create a POST object with policy condition "v2":
+        |key        |value                    |
+        |condition  |{"acl":"public-read"}    |
+        And I make a HTTP POST request with postObject "v2"
+        Then a file is uploaded with signature "v2" and its filename should be "filev2.ext"
+
+    Scenario: Upload an object via POST object with Signature V4
+        Given I have an s3 client and I have a file
+        When I create a POST object with policy condition "v4":
+        |key              |value              |
+        |acl              |public-read        |
+        |X-Amz-Algorithm  |AWS4-HMAC-SHA256   |
```

Comment:
Users should not have to specify the algorithm as part of their policy. That needs to be added by the signer.